standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -584,6 +584,32 @@ public OpenTxnsResponse openTxns(OpenTxnRequest rqst) throws MetaException {
           LOG.debug("Going to execute update <" + q + ">");
           stmt.execute(q);
         }
+
+        if (rqst.isSetReplPolicy()) {
+          List<String> rowsRepl = new ArrayList<>();
+          List<String> selectRepl = new ArrayList<>();
+
+          for (int i = 0; i < numTxns; i++) {
+            selectRepl.add("select TM_TARGET_TXN_ID from REPL_TXN_MAP where TM_REPL_POLICY = " + quoteString(
+                    rqst.getReplPolicy()) + "and TM_SRC_TXN_ID = " + rqst.getReplSrcTxnId().get(i));
+            long txnId = i + first;
+            rowsRepl.add(
+                quoteString(rqst.getReplPolicy()) + "," + rqst.getReplSrcTxnId().get(i) + "," + txnId);
+          }
+
+          List<String> queriesRepl = sqlGenerator.createInsertValuesStmt(
+              "REPL_TXN_MAP (TM_REPL_POLICY, tm_src_txn_id, TM_TARGET_TXN_ID)", rowsRepl);
```

Comment:
Mixed use of upper and lower case. Can be uniform.

Reply:
done